projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -321,6 +368,24 @@ public function load_feed( $force_refresh = false ) {
 		return $rss;
 	}
 
+	/**
+	 * Filter to override the default number of seconds to cache RSS feed data for the current feed.
+	 * Note that we don't use the feed's URL because some of the SimplePie feed caches trigger this
+	 * filter with a feed identifier and not a URL.
+	 *
+	 * @param int $cache_timeout_in_seconds Number of seconds to cache the podcast feed.
+	 *
+	 * @return int The number of seconds to cache the podcast feed.
+	 */
+	public function filter_podcast_cache_timeout( $cache_timeout_in_seconds ) {
+		if ( $this->cache_timeout !== null ) {
+			// If the cache timeout has already been reduced, return the smaller value of the two.
+			return min( $cache_timeout_in_seconds, $this->cache_timeout );
```

Comment:
Using `min` here is surprising to me--I would expect the value I provided to the `jetpack_podcast_feed_cache_timeout` filter to be used if set, not to only be used if it happens to be smaller than the current timeout.

If keeping it this way, the the doc comment should indicate the filter value is conditionally used.

Reply:
Along with keeping the `null` value as default, I have updated the code to always apply the value if it has been specified. I think we need both pieces, because it means we can assume the user specifically wants a different cache timeout for podcast feeds, because we wouldn't be in this code if the user left the podcast cache timeout as null.